.github/scripts/pr_summarizer.py

```
@@ -0,0 +1,63 @@
+1. Résumé clair et concis des changements (3-5 phrases max).
+2. Tableau récapitulatif avec colonnes : *Fichier* | *Résumé des changements*.
+3. Diagramme de séquence Mermaid (si pertinent) représentant les flux ou appels modifiés.
+4. Liste des points d’attention / risques potentiels pour le reviewer.
+5. (Optionnel) Idées d’améliorations ou de tests supplémentaires.
+
+### Pull Request Diff :
+{diff_text}
+"""
+
+# --- Appel API Mistral ---
+response = requests.post(
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)